pkgs/by-name/se/searxng/package.nix

```diff
@@ -8,13 +8,13 @@
 python3.pkgs.toPythonModule (
   python3.pkgs.buildPythonApplication rec {
     pname = "searxng";
-    version = "0-unstable-2024-11-25";
+    version = "0-unstable-2024-01-10";
```

Comment:
Can you double check the date? Isn't 2025 instead of 2024 ?

Reply:
ah. yes, right. thanks!